packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -147,9 +152,16 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		);
 	}
 
+	const filteredWrapperProps = applyFilters(
```

Comment:
This approach suffers from the hooks rules errors as filters get added/removed over time. I think it's probably going to create a lot of bugs.